client/src/components/MultiJurisdictionAudit/StatusBox/index.tsx

```diff
@@ -309,9 +309,8 @@ export const JurisdictionAdminStatusBox = ({
       )
 
     const numCompleted = auditBoards.filter(
-      ({ currentRoundStatus }) =>
-        currentRoundStatus.numAuditedBallots ===
-        currentRoundStatus.numSampledBallots
+      ({ currentRoundStatus, signedOffAt }) =>
+        currentRoundStatus.numSampledBallots === 0 || signedOffAt
```

Comment:
is this the right logic? `numSampledBallots === 0`? Or should it be `numAuditedBallots === numSampledBallots` ?

Reply:
if an audit board doesn't have any ballots assigned, the audit logic considers it "complete" because there's no work to be done. but it won't have `signedOffAt` populated. we account for this already everywhere but here